### PR TITLE
Implement new UI for Login page

### LINKED ISSUE #244: Implement new account start page
Here is a design idea for the new account start page inspired by Twitter and Google Assistant. Please implement it.

Ensure the bottom bar is the same as on https://chat.susi.ai/overview and the top bar the same as on https://skills.susi.ai

screenshot from 2018-05-25 23-13-05

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #244 

Changes:
- Implemented the new UI for Login page
- Added Responsiveness for mobile view.
- Removed Gif from older commit
- Fix position on Logo of top and bottom bar.
- Correct position of various import statements. 
- Remove scroll by adding responsiveness.

Surge Deployment Link: http://present-believe.surge.sh/

Screenshots for the change: 
login page

### CODE REVIEW

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -206,67 +233,101 @@ class Login extends Component {
...
+
+				        <RaisedButton
+				        label="Login"
+				        type="submit"
+				        backgroundColor={
+				        UserPreferencesStore.getTheme() === 'light' ? '#4285F4' : '#4285F4'}
+				        labelColor="#fff"
+				        disabled={!this.state.validForm} />
+				      </div>
+				      <br />
```
Comment (anshumanv) [inline src/components/Auth/Login/Login.react.js]: Please avoid using `br` tag, use css to achieve desired results.

**Comment by mariobehling** [comment]
Thanks, but as you can already see in your own screenshot many things aren't as expected, e.g.
* Top bar logo alignment and mein drop-down
* Bottom bar not same as in chat.susi.ai
* No responsive design
* Other issues pointed out above

The quality of this commit is very low. What are your next steps? How will you improve?

**Comment by mariobehling** [comment]
Actually top bar and bottom bar can simply be copied from other components. Not sure why you don't do this.

**Comment by PrP-11** [comment]
@DNS-404 I have enhanced the logo, it looks fine for now. Also minor CSS  enhancements can be made in subsequent PR. Thanks.

**Comment by PrP-11** [comment]
@mariobehling Since many changes like top and bottom bar have been implemented from other PRs therefore I am closing this PR and creating a new PR with only login page implementations. Thanks.

**Pull request closed without merging**